src/CommunityToolkit.Maui/Layouts/UniformItemsLayout.shared.cs

```diff
@@ -37,7 +37,7 @@ public int MaxRows
 	public int MaxColumns
 	{
 		get => (int)GetValue(MaxColumnsProperty);
-		set => SetValue(MaxColumnsProperty, value);
+		set => SetValue(MaxColumnsProperty, Math.Clamp(value, 1, int.MaxValue));
```

Comment:
I prefer we throw an exception when the user sets the value of `MaxColumns` below `1`.

This ensures that we educate the developer on the range of acceptable value.

If we use use `Math.Clamp`, then the user will try setting its value to `0` or `-1` and not see their desired behavior. They'll then try troubleshooting this bug, wasting their time, and may open a Bug Issue on our repo, wasting our time explaining that it's the expected behavior. 

Yes this exception will crash their app. But it is an Exception that the developer will only see once while testing their app and will quickly + easily fix after reading the `ArgumentOutOfRangeException` message.

```suggestion
		set => value >= 1 : SetValue(MaxRowsProperty, value) ? throw new new ArgumentOutOfRangeException(nameof(value), $"{nameof(MaxColumns)} must be greater or equal to {1}");
```